.github/release-to-wiki.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+GIT_REPOSITORY_URL="https://${GITHUB_PERSONAL_ACCESS_TOKEN}@github.com/$GITHUB_REPOSITORY.wiki.git"
+
+tmp_dir=$(mktemp -d -t ci-XXXXXXXXXX)
+(
+    cd "$tmp_dir" || exit 1
+    git init
+    git config user.name "$GITHUB_ACTOR"
+    git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+    git pull "$GIT_REPOSITORY_URL"
```

Comment:
hmm, would that be faster? I'll try changing.

Reply:
I don't know if it would be _faster_, but it wouldn't require the init command (no code _is_ faster than no code 😉 ). I've never seen it done this way before so partially a bit curious here myself 😄

Reply:
Possibly done this way so that it also works for private repos (since you add credentials before pulling)? I agree its cleaner to use clone so I changed it.

Reply:
Maybe? 🤔 I don't think user name/email are used for authentication though.